src/Products/Annotation/Util/AnnotationMapper.cs

```diff
@@ -24,18 +29,18 @@ private AnnotationMapper()
         /// <param name="annotations">AnnotationInfo[]</param>
         /// <param name="pageNumber">int</param>
         /// <returns></returns>
-        public AnnotationDataEntity[] mapForPage(AnnotationInfo[] annotations, int pageNumber)
+        public AnnotationDataEntity[] MapForPage(AnnotationBase[] annotations, int pageNumber, PageInfo pageInfo)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'MapForPage' a static method.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)